CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Snyk Security - Code and Open Source Dependencies Changelog
 
+## [1.26.1]
+
```

Comment:
I'm not sure about this version 🤔  Where can I validate it?

Reply:
It should be the next version (minor) after the currently released.